packages/editor/src/package/index.ts

```diff
@@ -7,6 +7,8 @@ export { defaultPlugins } from './config'
 export type { BaseEditor } from '@editor/core'
 
 export { EditorPluginType } from '@editor/types/editor-plugin-type'
+export { TemplatePluginType } from '@editor/types/template-plugin-type'
```

Comment:
Sorry if I'm repeating myself, but could you please explain why integrations will need this now?

Reply:
Sorry, forgot to remove it. It's not needed.